pylhc/constants/machine_settings_info.py

```diff
@@ -11,13 +11,20 @@
 # Filename
 info_name = f"machine_settings{TFS_SUFFIX}"
 knobdef_suffix = f"_definition{TFS_SUFFIX}"
+trimhistory_suffix = f"_trims{TFS_SUFFIX}"
 
 # Columns
 column_knob = "KNOB"
+column_time = "TIME"
+column_timestamp = "TIMESTAMP"
 column_value = "VALUE"
 
+
 # Headers
+head_accel = "Accelerator"
 head_time = "Time"
+head_start_time = "StartTime"
```

Comment:
very nitpicky, but for consistency, shouldn't all headers follow the same scheme? so BP_Start( also Optics_Start) should then be BPStart or maybe even StartBP?

Reply:
I think I kept it, because that are the names that came from pjlsa. but actually you are right. I will unify this to CamelCase